### ISSUE TITLE
Can't pair Samsung TV

### ISSUE THREAD

**Issue by Ioan-Popovici** [tavicu/homebridge-samsung-tizen#307]
I have a Samung QLED with Tizen 4.0 but the allow message does not appear.
My mac ip addresses in the config are correct.

I get the following error message:
Failed to pair! Make sure TV is online and you click "Allow" on the popup

Home Assistant for example is detecting it automatically, but homebridge can't connect to it.
(I have both HA and HB)

**Comment by tavicu** [comment]
If you migrated from Home Assistant to Homebridge try to change the name of TV in the config.
Also you can try adding the UUID setting if you are not able to add the TV to home app. https://github.com/tavicu/homebridge-samsung-tizen/issues/245#issuecomment-691895984

I need more details than this to understand what you did and if it worked on Home Assistant. Don't understand the "detecting" since the plugin does not detect anything automatically!

**Comment by Ioan-Popovici** [comment]
I haven't migrated, HA somehow autodetects the airplay2 feature of the TV or I don't understand something.
This is my model: Samsung QE49Q6FNA

I've tried to add it in HA but it says that this TV version is not supported, so I was mistaken, it does not work in HA either.

Can I get you some logs?

**Comment by Ioan-Popovici** [comment]
So a more lengthy explanation:

I've installed HA because it seemed a better option than HB for my shelly switches. In the end I flashed my shelly devices with homekit firmware so I just kept HA for AdGuard (ad blocking).

I could jot get my dyson fan and my roborock vacuum working with HA so I installed HB in a container side by side with HA.

After I got my Dyson fan and vacuum cleaner working trough HB I wanted to add my Samsung TV to HB but as I said it does not seem to be able to initiate the pairing request.

Long story short I am a bit new to this but I have good network knowledge and decent scripting experience. Also I only use HA for ad filtering and nothing more for now.

**Comment by Ioan-Popovici** [comment]
I tried the UUID method but it still doesn't want to connect.

**Comment by Ioan-Popovici** [comment]
Update:

After the UUID method it took about 3 minutes after homebridge restart, afte

**Comment by Ioan-Popovici** [comment]
Ok, it shows up in the dashboard but not in homekit, and I get this error.

Failed to open socket {"event":"ms.channel.timeOut"}

**Comment by Ioan-Popovici** [comment]
I give up... I tried everything. I can't get the popup anymore, even though I deleted all the authorizations from the TV.

My Config:
```
{
            "name": "SamsungTizen",
            "devices": [
                {
                    "name": "Living Room TV",
                    "ip": "192.168.111.11",
                    "mac": "64:2C:AB:6E:7C:DA",
                    "uuid": "A1"
                }
            ],
            "platform": "SamsungTizen"
        }
```

**Comment by tavicu** [comment]
Scrie-mi pe FB messenger (Octavian Ruda) si vedem luni cum te pot ajuta. Eventual iti dau request-u de websocket sa vedem daca poate fi accessat. Pare ca ai cunostintele necesare pentru a face nitel debug :)

O sa sterg in 30 minute mesaju de aici!

**Comment by tavicu** [comment]
Ti-am scris pe twitter

**Comment by Ioan-Popovici** [comment]
Doesn't seem to work when homebridge is on a different VLAM despite IGMP Snooping being enabled.
Seems like a websocket problem not related to the plugin.
I'll dig around more to find the issue.
Thank you,
Ioan

**Comment by adamtharani** [comment]
@Ioan-Popovici Did you have any luck figuring this out? I'm running into the same issue across VLANS

**Comment by Ioan-Popovici** [comment]
No, I've moved the tv to the main vlan.
I'm getting a ir blaster anyway and that will fix the problem
